package/libskylark/Config.in

```diff
@@ -0,0 +1,8 @@
+config BR2_PACKAGE_LIBSKYLARK
+  bool "libskylark"
+  select BR2_PACKAGE_LIBCURL
+  select BR2_PACKAGE_CZMQ
```

Comment:
It doesn't seem like you need this package anymore - this library doesn't seem to do any ZMQ stuff anymore (which is great). Ditto for LIBSBP_ZMQ below?